core/src/main/scripts/import_data_validator/validateData.py

```diff
@@ -1422,13 +1427,139 @@ class SegValidator(Validator):
         'seg.mean']
     REQUIRE_COLUMN_ORDER = True
 
+    def __init__(self, *args, **kwargs):
+        """Initialize validator to track coverage of the genome."""
+        super(SegValidator, self).__init__(*args, **kwargs)
+        self.chromosome_lengths = self.load_chromosome_lengths(
+            self.meta_dict['reference_genome_id'])
+
```

Comment:
remember to also validate the value of this field reference_genome_id. I think you are not validating it anywhere. It should be hg19 for now. Other values should be blocked. Do we also have other fields that should be validated to be hg19?